src/app/components/ResultsCount/ResultsCount.jsx

```diff
@@ -87,40 +98,36 @@ class ResultsCount extends React.Component {
     const plural = (searchKeywords && searchKeywords.indexOf(' ') !== -1) ? 's' : '';
 
     if (isLoading) {
-      return (<p>Loading...</p>);
+      return ('Loading...');
     }
 
     if (count !== 0) {
-      return (<h2>Displaying {currentResultDisplay} of {countF} results {displayContext}</h2>);
+      return (`Displaying ${currentResultDisplay} of ${countF} results ${displayContext}`);
     }
 
     if (this.checkSelectedFilters()) {
       return (
-        <h2>No results for the keyword{plural} "{searchKeywords}" with the chosen filters. Try
-          a different search or different filters.
-        </h2>
+        `No results for the keyword${plural} "${searchKeywords}" with the chosen filters. Try a different search or different filters.`
       );
     }
 
-    return (
-      <h2>
-        No results for the keyword{plural} "{searchKeywords}". Try a different search.
-      </h2>
-    );
+    return `No results for the keyword${plural} "${searchKeywords}". Try a different search.`;
   }
 
```

Comment:
Nice catch removing presentation code from the messages returned by ResultsCount.